src/renderer/component/shapeShift/internal/form.jsx

```diff
@@ -49,23 +49,27 @@ export default (props: Props) => {
   } = props;
   return (
     <form onSubmit={handleSubmit}>
-      <div className="form-field">
-        <span>{__('Exchange')} </span>
-        <select
-          className="form-field__input form-field__input-select"
-          name="originCoin"
-          onChange={e => {
-            getCoinStats(e.target.value);
-            handleChange(e);
-          }}
-        >
-          {shiftSupportedCoins.map(coin => (
-            <option key={coin} value={coin}>
-              {coin}
-            </option>
-          ))}
-        </select>
-        <span> {__('for LBC')}</span>
+      <FormRow
```

Comment:
I think I'm going to change this to `FormElement` and keep `FormRow` which just allows several `FormElement`s side by side